composer.json

```diff
@@ -27,7 +27,7 @@
     },
     "require": {
         "ext-dom": "*",
-        "bower-asset/bootstrap-datepicker": "^1.10",
+        "bower-asset/bootstrap-datepicker": "^1.9",
```

Comment:
Composer won't be able to update without this version change. Not sure why.

Reply:
I ran into this same issue recently in a local branch. It seems that bower removed the 1.10 release for some reason. In my local branch I got it working by switching from `bower-asset/bootstrap-datepicker` to `npm-asset/bootstrap-datepicker` which still has the 1.10 release.

Reply:
Thanks for the suggestion. I've replaced the module.